Libraries/LibCompress/Deflate.cpp

```diff
@@ -0,0 +1,428 @@
+/*
+ * Copyright (c) 2020, the SerenityOS developers
```

Comment:
Feel free to use your actual name, be proud of your contribution! :)
(Also, in case of $LegalStuff, it might help to know who the actual Copyright holders are.)

Reply:
I'd prefer not to :)